cpapp_backend/cpapp/services/api_client.py

```
@@ -38,29 +38,75 @@ def _make_request(self, method: str, endpoint: str, params: Optional[Dict[str, A
+                    "error": response_text,
+                    "url": url,
+                    "method": method
+                }
+                logger.error(f"HTTP error {response.status_code}: {response_text}")
+                return error_response
             
+            # Try to parse JSON response
             try:
-                return response.json()
+                json_response = response.json()
+                logger.debug(f"Successfully parsed JSON response")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)